stytch/b2b/passwords/discovery/email/types.go

```diff
@@ -0,0 +1,93 @@
+	PasswordResetToken string `json:"password_reset_token,omitempty"`
+	// Password: The password to authenticate, reset, or set for the first time. Any UTF8 character is allowed,
+	// e.g. spaces, emojis, non-English characers, etc.
+	Password         string `json:"password,omitempty"`
+	PkceCodeVerifier string `json:"pkce_code_verifier,omitempty"`
+}
+
+// ResetStartParams: Request type for `Email.ResetStart`.
+type ResetStartParams struct {
+	// EmailAddress: The email address of the Member to start the email reset process for.
+	EmailAddress string `json:"email_address,omitempty"`
+	// ResetPasswordRedirectURL: The URL that the Member clicks from the reset password link. This URL should
+	// be an endpoint in the backend server that verifies the request by querying
+	//   Stytch's authenticate endpoint and finishes the reset password flow. If this value is not passed, the
+	// default `reset_password_redirect_url` that you set in your Dashboard is used.
+	//   If you have not set a default `reset_password_redirect_url`, an error is returned.
+	ResetPasswordRedirectURL string `json:"reset_password_redirect_url,omitempty"`
+	// DiscoveryRedirectURL: The URL that the end user clicks from the discovery Magic Link. This URL should be
+	// an endpoint in the backend server that
+	//   verifies the request by querying Stytch's discovery authenticate endpoint and continues the flow. If
+	// this value is not passed, the default
+	//   discovery redirect URL that you set in your Dashboard is used. If you have not set a default discovery
+	// redirect URL, an error is returned.
+	DiscoveryRedirectURL string `json:"discovery_redirect_url,omitempty"`
+	// ResetPasswordTemplateID: Use a custom template for reset password emails. By default, it will use your
+	// default email template. The template must be a template using our built-in customizations or a custom
+	// HTML email for Magic Links - Reset Password.
+	ResetPasswordTemplateID string `json:"reset_password_template_id,omitempty"`
+	// ResetPasswordExpirationMinutes: Sets a time limit after which the email link to reset the member's
+	// password will no longer be valid.
+	ResetPasswordExpirationMinutes int32  `json:"reset_password_expiration_minutes,omitempty"`
+	PkceCodeChallenge              string `json:"pkce_code_challenge,omitempty"`
+	// Locale: Used to determine which language to use when sending the user this delivery method. Parameter is
+	// a [IETF BCP 47 language tag](https://www.w3.org/International/articles/language-tags/), e.g. `"en"`.
+	//
+	// Currently supported languages are English (`"en"`), Spanish (`"es"`), and Brazilian Portuguese
+	// (`"pt-br"`); if no value is provided, the copy defaults to English.
+	//
+	// Request support for additional languages
+	// [here](https://docs.google.com/forms/d/e/1FAIpQLScZSpAu_m2AmLXRT3F3kap-s_mcV6UTBitYn6CdyWP0-o7YjQ/viewform?usp=sf_link")!
+	//
+	Locale string `json:"locale,omitempty"`
```

Comment:
This shouldn't be here any more, did you run sync prior to generating these?

Reply:
Think this is correct - there's an identically-named type in a different package that had `locale` removed from it, but this one aligns with the protos.